laze-project.yml

```
@@ -180,28 +180,38 @@ contexts:
         - ${SCRIPTS}/debug-openocd.sh
       OPENOCD_ARGS:
         - "-f board/nordic_nrf52_dk.cfg"
-      PROBE_RS_CHIP: nrf52832_xxAA
-
-  - name: nrf5340
-    parent: nrf
-    selects:
-      - thumbv8m.main-none-eabi # actually eabihf, but ariel-os doesn't support hard float yet
-    env:
-      PROBE_RS_CHIP: nrf5340_xxAA
 
   - name: nrf52832
     parent: nrf52
+    selects:
+      - cortex-m4f
+    env:
+      PROBE_RS_CHIP: nrf52832_xxAA
 
   - name: nrf52833
     parent: nrf52
+    selects:
+      - cortex-m4f
     env:
       PROBE_RS_CHIP: nrf52833_xxAA
 
   - name: nrf52840
     parent: nrf52
+    selects:
+      - cortex-m4f
     env:
       PROBE_RS_CHIP: nrf52840_xxAA
 
+  - name: nrf53
+    parent: nrf
+
+  - name: nrf5340
+    parent: nrf53
+    selects:
+      - cortex-m4
```

this was `thumbv8m.main-none-eabi` before, so `cortex-m33f` probably?

```suggestion
      - cortex-m33f
```

---

Good catch! And indeed it has an FPU, as we're currently targeting the application core (the network core doesn't have one).